--- a/gettext-tools/src/Makefile.am
+++ b/gettext-tools/src/Makefile.am
@@ -115,7 +115,8 @@ AM_CPPFLAGS = \
   -I$(top_srcdir)/tree-sitter-$(TREE_SITTER_VERSION)/lib/include \
   -I../libgrep -I$(top_srcdir)/libgrep \
   -I../gnulib-lib -I$(top_srcdir)/gnulib-lib \
-  -I../../gettext-runtime/intl -I$(top_srcdir)/../gettext-runtime/intl
+  -I../../gettext-runtime/intl -I$(top_srcdir)/../gettext-runtime/intl \
+  -DBISON_LOCALEDIR=\"@BISON_LOCALEDIR@\"
 DEFS = \
   -DLOCALEDIR=$(localedir_c_make) \
   -DUSEJAVA=$(USEJAVA) \
